src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/StormLcmSpec.groovy

```diff
@@ -67,8 +67,8 @@ class StormLcmSpec extends HealthCheckSpecification {
         then: "Database nodes and relations are unchanged"
         def newRelation = database.dumpAllRelations()
         def newSwitches = database.dumpAllSwitches()
-        expect newSwitches, sameBeanAs(switchesDump)
-        expect newRelation, sameBeanAs(relationsDump).ignoring("time_modify").ignoring("latency")
+        expect newSwitches, sameBeanAs(switchesDump).ignoring("data.timeModify").ignoring("data.features")
```

Comment:
Why ignoring 'features'? You may be masking a real problem here. We had a defect related to 'features' inconsistency in database switch objects after restarting topologies (related to mfl patch), but it should be fixed in the latest develop

Reply:
Good catch. No need to ignore the features. I removed it.